components/CheckDocumentation/vacuum/UnnecessaryTableGrowth.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+
+import {
+  CheckDocs,
+  CheckGuidanceProps,
+  CheckTriggerProps,
+} from "../../../util/checks";
+import { formatBytes } from "../../../util/format";
+
+const UnnecessaryTableGrowthTrigger: React.FunctionComponent<
+  CheckTriggerProps
+> = ({ config }) => {
+  return (
+    <p>
+      Detects when a table has experienced unnecessary growth due to
+      insufficient VACUUM. It analyzes the table statistic from the past seven
+      days and calculates the amount of growth that could have been avoided if
+      VACUUM had been performed more frequently. Creates an issue when the
+      unnecessary growth exceeds{" "}
+      <code>{formatBytes(Number(config.settings["notify_bytes"]))}</code>, as
+      well as <code>{config.settings["notify_pct"]}%</code> of the current size.
+      Resolves once the unnecessary growth decreases due to changes in
+      autovacuum settings or table usage.
+    </p>
+  );
+};
+
+const UnnecessaryTableGrowthGuidance: React.FunctionComponent<
+  CheckGuidanceProps
+> = () => {
+  // UnnecessaryTableGrowth uses a similar pattern as MissingIndex.
+  // The docs doesn't have the solution section as it's in dynamic guidance generated in app side.
+  return (
+    <div>
+      <h4>Impact</h4>
+      <p>
+        In Postgres, any DELETEs or UPDATEs create dead rows that can be
+        VACUUMed to become reusable space for future INSERTs or UPDATEs.
+        Autovacuum is designed to automatically perform VACUUM regularly to
+        efficiently reclaim reusable space created by removing dead rows. This
+        helps ensure that INSERTs and UPDATEs can use existing space instead of
+        claiming new space. In this way, the table can avoid unnecessary growth.
```

Comment:
I think we should emphasize the problem more strongly before talking about the mitigation. How about something like this?

```suggestion
        In Postgres, any DELETEs or UPDATEs create dead rows in the tables they modify.
        VACUUM allows Postgres to reclaim these rows and reuse that space for future INSERTs or UPDATEs.
        Autovacuum is designed to perform VACUUM regularly in order to
        efficiently reuse space. This
        helps ensure that INSERTs and UPDATEs can use existing space instead of
        claiming new space. In this way, the table can avoid unnecessary growth.
```

I also reworded a bit for conciseness and to avoid reusing "reclaim" (which I think is more useful in the second sentence).

Reply:
👍 I'll tweak "reclaim these rows" to "reclaim these dead rows" (I was able to read smoother with it as I didn't need to think about what "these rows" were)